### ISSUE TITLE
Gems 0.12.0 to 0.32.0 are not published

### ISSUE THREAD

**Issue by blowmage** [googleapis/gapic-generator-ruby#1079]
The gems published to RubyGems stopped at version 0.11.0. The gems versions 0.12.0 to 0.32.0 were never published. To use the latest generator version I had to pull the repo and build the gems locally.

Compare the published gem versions: https://rubygems.org/gems/gapic-generator/versions
To the tagged versions in the repo: https://github.com/googleapis/gapic-generator-ruby/tags

**Comment by dazuma** [comment]
Yeah we haven't been pushing out gem releases since the internal generation process no longer uses them. We also don't have up to date instructions for performing codegen using the released gem.

**Comment by dazuma** [comment]
The release process is now publishing gems again.
